setup.py

```diff
@@ -241,6 +241,7 @@ def check_bin(ccbin, source, allow_dash):
         "scipy >= 1.9.0",
         "h5py >= 3.6.0",
         "hdmedians >= 0.14.1",
+        "statsmodels >= 0.10.0"
```

Comment:
Given that we had consensus in the previous discussion, how about dropping this line and adding a `_benjamini_hochberg` helper function [along with this](https://github.com/scikit-bio/scikit-bio/blob/72104932830ace439ed64c298ce354264e23c75c/skbio/stats/composition.py#L1530)?

Reply:
I've removed the statsmodels dependency. I've decided to use the `_holm_boniferroni` correction instead, since it is already implemented in skbio